Makefile

```diff
@@ -156,6 +156,11 @@ ifneq (,$(findstring openldap,$(ENABLED_DOCKER_SERVICES)))
 endif
 endif
 
+run-haserver: run-client
+	@echo Starting docker containers in HA formation
+
```

Comment:
We should add a check here for `ifeq ($(BUILD_ENTERPRISE_READY),true)`, and only run this in that case. Mainly to help contributors not run it accidentally and have it not work.

Reply:
totally

Reply:
I think it should be `BUILD_ENTERPRISE_READY`. `BUILD_ENTERPRISE` is just an env var to control whether to build enterprise or not. The `BUILD_ENTERPRISE_READY` checks the existence of the `enterprise` directory which is more robust and used elsewhere in the Makefile.

Reply:
did i paste the wrong thing?... I did. Sorry about that.

Reply:
Hmm .. I still don't see the changes though :thinking: Not sure if I am missing something.

Reply:
haven't pushed them yet 😄 , battling with Cypress atm

Reply:
Ah I see. When you requested my review, I thought the changes were there. No worries, just give me a ping when ready.